docs/code-examples/roundtrips.py

```diff
@@ -56,6 +56,8 @@
 
 # fmt: on
 
+cpp_build_dir = "./build/roundtrips"
```

Comment:
Can we make these paths relative to the location of the script itself while we're at it?

Reply:
I actually intentionally wanted to share the folder with the other roundtrip test. But that's technically just because I "happen to know" they have the same cmake config command, saving me as a local tester time. If we put them local I have to build libarrow again for each, albeit only once so maybe that's not a strong argument 🤔 

The other thing is that so far we keep target/build style of folder at top level: I found it nice that killing your build folder resets the entire C++ build

Reply:
> The other thing is that so far we keep target/build style of folder at top level

Yes that's what I meant: make sure this script always does its stuff at the root of the workspace, no matter what the CWD of the caller happens to be.